libs/clients/radom/radom.go

```diff
@@ -0,0 +1,169 @@
+package radom
+
+import (
+	"context"
+	"crypto/subtle"
+	"errors"
+	"time"
+
+	"github.com/shopspring/decimal"
+
+	"github.com/brave-intl/bat-go/libs/clients"
+)
+
+var (
+	ErrInvalidMetadataKey = errors.New("invalid metadata key")
+)
+
+// CheckoutSessionRequest represents a request to create a checkout session.
+type CheckoutSessionRequest struct {
+	SuccessURL     string                 `json:"successUrl"`
+	CancelURL      string                 `json:"cancelUrl"`
+	Currency       string                 `json:"currency"`
+	ExpiresAt      int64                  `json:"expiresAt"` // in unix seconds
```

Comment:
What do we set this to when we create a Radom session?

Reply:
these are set from the sku values, and the success/cancel urls are the brace account site, currency is BAT and expires at is the end of the sku duration.  here is an example of what we set:

https://github.com/brave-intl/bat-go/pull/1789/files#diff-30be4362fa150d72b34eede7fddf4e7bc71aa79064ae0dc514bc311b613ab19aR11-R19